chat-plugins/economy.js

```diff
@@ -1,270 +1,362 @@
+/*
+* economy.js by CreaturePhil
+* Dice Game Credits - SilverTactic (Silveee) refactored by Lord Haji
+*/
 'use strict';
 
-SG.database = new sqlite3.Database('config/users.db', function () {
-	SG.database.run("CREATE TABLE if not exists users (userid TEXT, name TEXT, currency INTEGER, lastSeen INTEGER, onlineTime INTEGER, credits INTEGER, title TEXT, notifystatus INTEGER, background TEXT)");
-	SG.database.run("CREATE TABLE if not exists friends (id integer primary key, userid TEXT, friend TEXT)");
-});
```

Comment:
DO NOT REMOVE THE SQLITE3 YET. Depreciated means soon to remove, we need to keep the sqlite3 data untill we can extract it to the new database so we dont reset everyones currency ect.